Rakefile

```diff
@@ -46,6 +46,37 @@ namespace :test do
     bcl = BCL::ComponentMethods.new
     bcl.login
   end
+
+  desc 'test component spreadsheet'
+  task :spreadsheet do
+    bclcomponents = BCL::ComponentFromSpreadsheet.new(File.expand_path('../lib/files/Components.xls', __FILE__), ['Roofing'])
+    bclcomponents.save(File.expand_path('../lib/files/staged', __FILE__))
+  end
+
+  desc 'test measure download'
```

Comment:
@kflemin this feels like it should be a test instead of a rake task

Reply:
these are more examples vs. tests.